lib/taskana-core/src/main/java/pro/taskana/spi/priority/internal/PriorityServiceManager.java

```diff
@@ -34,6 +34,14 @@ public boolean isEnabled() {
   }
 
   public OptionalInt calculatePriorityOfTask(TaskSummary task) {
+    if (task.getManualPriority() >= 0) {
```

Comment:
I prefer encapsulating this logic within the Task itself instead of implicitly using this everywhere where it's needed.
```
if(task.hasManualPriority()) {
...
}
```
is more readable to me.

Reply:
This would be more "clean code style" then. I like your idea.